src/DssSpell.sol

```diff
@@ -51,328 +52,235 @@ contract DssSpellAction is DssAction {
-
-        // Remove RWA015-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("RWA015-A");
-
-        // Remove TUSD-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("TUSD-A");
-
-        // Remove UNI-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("UNI-A");
-
-        // Remove UNIV2AAVEETH-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("UNIV2AAVEETH-A");
-
-        // Remove UNIV2DAIETH-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("UNIV2DAIETH-A");
-
-        // Remove UNIV2DAIUSDT-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("UNIV2DAIUSDT-A");
-
-        // Remove UNIV2ETHUSDT-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("UNIV2ETHUSDT-A");
-
-        // Remove UNIV2LINKETH-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("UNIV2LINKETH-A");
-
-        // Remove UNIV2UNIETH-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("UNIV2UNIETH-A");
-
-        // Remove UNIV2USDCETH-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("UNIV2USDCETH-A");
-
-        // Remove UNIV2WBTCDAI-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("UNIV2WBTCDAI-A");
-
-        // Remove UNIV2WBTCETH-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("UNIV2WBTCETH-A");
-
-        // Remove USDC-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("USDC-A");
-
-        // Remove USDC-B from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("USDC-B");
-
-        // Remove USDT-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("USDT-A");
-
-        // Remove YFI-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("YFI-A");
-
-        // Remove ZRX-A from the ilk registry
-        IlkRegistryAbstract(ILK_REGISTRY).removeAuth("ZRX-A");
-
-        // Note: Bump Chainlog version to 1.20.3 as periphery contracts are being removed
-        DssExecLib.setChangelogVersion("1.20.3");
-
-        // ----- Execute Spark Proxy Spell -----
-        // Forum: https://forum.sky.money/t/august-7-2025-proposed-changes-to-spark-for-upcoming-spell/26896
-        // Poll: https://vote.sky.money/polling/QmXLExe7
-        // Poll: https://vote.sky.money/polling/QmVGr47c
-        // Poll: https://vote.sky.money/polling/QmUevv3W
-        // Poll: https://vote.sky.money/polling/QmU6L1gS
-        // Poll: https://vote.sky.money/polling/QmZu3tVL
-
-        // Execute Spark proxy spell at 0xb12057500EB57C3c43B91171D52b6DB141cCa01a
+        // Execute Spark proxy spell at 0xa57d3ea3aBAbD57Ed1a1d91CD998a68FB490B95E
         ProxyLike(SPARK_PROXY).exec(SPARK_SPELL, abi.encodeWithSignature("execute()"));
```

Comment:
There are Forum links missing for the following items from the Spark spell content (which is different from the ones that come before it):
```
[Ethereum] Spark Treasury - Transfer USDS to Spark Foundation
[Ethereum] Spark Liquidity Layer - Update SparkLend USDS Rate Limits
[Ethereum] Spark USDS Morpho Vault - Adjust PT-USDS-SPK-25Sept Pool Supply Cap
```